src/components/navBar.tsx

```diff
@@ -33,28 +35,111 @@ export default function NavBar() {
           variant="filled"
           colorScheme="b4"
           aria="sign up"
-          onClick={handleSignupClick}
+          value="signup"
+          onClick={handleNavButtonClick}
         >
           sign up
         </SiteButton>
+
         <Link href={"/support"}>
-          <SiteButton variant="filled" colorScheme="e5" aria="donate">
+          <SiteButton
+            variant="filled"
+            colorScheme="e5"
+            aria="donate"
+            value="support"
+            onClick={handleNavButtonClick}
+          >
             support us
           </SiteButton>
         </Link>
-        <SiteButton variant="filled" colorScheme="f3" aria="share on socials">
+        <Link href={"/pricing"}>
+          <SiteButton
+            variant="filled"
+            colorScheme="d4"
+            aria="pricing"
+            value="pricing"
+            onClick={handleNavButtonClick}
+          >
+            pricing
+          </SiteButton>
+        </Link>
+        <SiteButton
+          variant="filled"
+          colorScheme="f3"
+          aria="share on socials"
+          value="share"
+          onClick={handleNavButtonClick}
+        >
           share
         </SiteButton>
       </div>
-      {signupClicked === true ? (
-        <div className="SignupButtons absolute right-64 top-24 flex flex-col items-start space-y-4">
-          <SiteButton variant="filled" colorScheme="b3" aria="business">
+      {clickedButton === "share" ? (
```

Comment:
I see now that you're using the state to display or hide drop downs. It would be much better to make a dropdown component and pass open as a boolean prop to this component

Reply:
BUT I'm looking into making the dropdown component 👍

Reply:
Alrighty - I've got different components made for those. I think it's what you're talking about? At any rate, I like it + it makes it easier to read + work with 😀